app/backend/src/couchers/resources.py

```diff
@@ -93,10 +93,22 @@ def copy_resources_to_database(session):
     Truncating and recreating guarantees the data is fully in sync.
     """
```

Comment:
Instead of deleting these here, do it in the migration. For two reasons:

1. Mainly this function is by nature a bit more involved with the deferred constraints, and so I'd like this to stay very clear and specialised in what it does (replacing the resources tables in the db)
2. It's good to be explicit in the migration as to what's going on.